tests/ui-api-layer-acceptance-tests/k8s/client.go

```diff
@@ -3,6 +3,7 @@ package k8s
 import (
 	scClientset "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
```

Comment:
In the meantime could you please rename the `k8s` package to `client`? It's definitely confusing to have both `k8s` in `domain` and in `root` folder. Thanks!